packages/module/src/AssistantMessageEntry/AssistantMessageEntry.tsx

```diff
@@ -41,20 +81,30 @@ export const AssistantMessageEntry = ({ children, options, icon: IconComponent =
           </TextContent>
         </SplitItem>
       </Split>
-      {options ? ( 
+      {options ? (
         <Split>
-          <SplitItem className={classnames(classes.chatbot,"pf-v5-u-ml-xl pf-v5-u-mt-md")}>
-            {options.map((option, index) => (
-              <Label key={index} className="pf-v5-u-m-xs" {...option.props}>
-                {option.title}
-              </Label>
-            ))}
+          <SplitItem className={classnames(classes.chatbot, "pf-v5-u-ml-xl pf-v5-u-mt-md")}>
+            {options.map((option, index) => {
+              const { onClick: customOnClick, ...restProps } = option.props || {};
+              return (
+                <Label
+                  key={index}
+                  className={classnames(classes.label, "pf-v5-u-m-xs pf-m-red", {
+                    [classes.activeOption]: selectedOptionIndex === index,
+                    [classes.disabledOption]: selectedOptionIndex !== null && selectedOptionIndex !== index
+                  })}
+                  onClick={() => handleOptionClick(index, customOnClick)}
+                  {...restProps}
```

Comment:
Lets move the `...restProps` before the `onClick` and `className`  so that it can't be overridden :thinking: 

Consider this: https://playcode.io/1948817